### PR TITLE
Changing owl version in all available languages

### PR DESCRIPTION
**Pull request by riccardoAlbertoni**
This PR updates and simplifies the owl:version in dcat.ttl addressing issues #979 

The proposed  owl:version in English is 
` owl:versionInfo """This is an updated copy of v2.0 of the DCAT vocabulary, taken from https://www.w3.org/ns/dcat.ttl"""@en `

The PR updates the Italian, Spanish, and Czech translations accordingly.

In case this PR is accepted, we might need to regenerate the other DCAT encodings  (jsonld and rdf), updating the files provided in  #1110.

### CODE REVIEW

**Review by davebrowning — APPROVED** [review]

File: dcat/rdf/dcat.ttl [hunk 1]
```diff
@@ -152,11 +152,10 @@
   owl:imports dct: ;
   owl:imports <http://www.w3.org/2004/02/skos/core> ;
   owl:imports <http://www.w3.org/ns/prov-o#> ;
-  owl:versionInfo """Esta es una copia del vocabulario DCAT v1.0 disponible en https://www.w3.org/ns/dcat.ttl
-con modificaciones en progreso para la nueva versión de DCAT que resultará del grupo 'W3C Data Exchange Working Group'"""@es ;
-  owl:versionInfo """This is an updated copy of v1.0 of the DCAT vocabulary, taken from https://www.w3.org/ns/dcat.ttl and will replace that file when DCAT2 is published
-Temporarily added imports for DCTerms, SKOS, FOAF to assist in development of v1.* or v2.0 by W3C Data Exchange Working Group"""@en ;
-  owl:versionInfo "Toto je aktualizovaná kopie slovníku DCAT verze 1.0, převzatá z https://www.w3.org/ns/dcat.ttl a tento soubor po vydání DCAT2 nahradí. Dočasně jsou přidány importy pro DCTerms, SKOS, FOAF pro pomoc ve vývojei v1.* či v2.0 skupinou W3C Data Exchange Working Group"@cs ;
+  owl:versionInfo """Esta es una copia del vocabulario DCAT v2.0 disponible en https://www.w3.org/ns/dcat.ttl"""@es ;
```
Comment (riccardoAlbertoni) [inline dcat/rdf/dcat.ttl]:
```suggestion
  owl:versionInfo "Esta es una copia del vocabulario DCAT v2.0 disponible en https://www.w3.org/ns/dcat.ttl"@es ;
```

File: dcat/rdf/dcat.ttl [hunk 2]
```diff
@@ -152,11 +152,10 @@
...
+  owl:versionInfo """This is an updated copy of v2.0 of the DCAT vocabulary, taken from https://www.w3.org/ns/dcat.ttl"""@en ;
```
Comment (riccardoAlbertoni) [inline dcat/rdf/dcat.ttl]:
```suggestion
  owl:versionInfo "This is an updated copy of v2.0 of the DCAT vocabulary, taken from https://www.w3.org/ns/dcat.ttl"@en ;
```

File: dcat/rdf/dcat.ttl [hunk 3]
```diff
@@ -152,11 +152,10 @@
...
+  owl:versionInfo """Questa è una copia aggiornata del vocabolario DCAT v2.0 disponibile in https://www.w3.org/ns/dcat.ttl"""@en ;
```
Comment (riccardoAlbertoni) [inline dcat/rdf/dcat.ttl]:
```suggestion
  owl:versionInfo "Questa è una copia aggiornata del vocabolario DCAT v2.0 disponibile in https://www.w3.org/ns/dcat.ttl"@en ;
```

File: dcat/rdf/dcat.ttl [hunk 4]
```diff
@@ -152,11 +152,10 @@
...
-con modificaciones en progreso para la nueva versión de DCAT que resultará del grupo 'W3C Data Exchange Working Group'"""@es ;
-  owl:versionInfo """This is an updated copy of v1.0 of the DCAT vocabulary, taken from https://www.w3.org/ns/dcat.ttl and will replace that file when DCAT2 is published
-Temporarily added imports for DCTerms, SKOS, FOAF to assist in development of v1.* or v2.0 by W3C Data Exchange Working Group"""@en ;
-  owl:versionInfo "Toto je aktualizovaná kopie slovníku DCAT verze 1.0, převzatá z https://www.w3.org/ns/dcat.ttl a tento soubor po vydání DCAT2 nahradí. Dočasně jsou přidány importy pro DCTerms, SKOS, FOAF pro pomoc ve vývojei v1.* či v2.0 skupinou W3C Data Exchange Working Group"@cs ;
+  owl:versionInfo "Esta es una copia del vocabulario DCAT v2.0 disponible en https://www.w3.org/ns/dcat.ttl"@es ;
+  owl:versionInfo "This is an updated copy of v2.0 of the DCAT vocabulary, taken from https://www.w3.org/ns/dcat.ttl"@en ;
+  owl:versionInfo """Questa è una copia aggiornata del vocabolario DCAT v2.0 disponibile in https://www.w3.org/ns/dcat.ttl"""@en ;
```
Comment (riccardoAlbertoni) [inline dcat/rdf/dcat.ttl]:
```suggestion
  owl:versionInfo "Questa è una copia aggiornata del vocabolario DCAT v2.0 disponibile in https://www.w3.org/ns/dcat.ttl"@en ;
```

**Comment by riccardoAlbertoni** [comment]
I have deleted the use of multiline literal for owl:version. 

I have done it in this PR, firstly to avoid conflict with PR #1127 and secondly, considering that the simplified text proposed in this PR, does not require a multiline literal anymore, so if we accept this PR we should get rid of the multiline independently from  PR #1127.

**Review by andrea-perego — APPROVED** [review]

**Pull request merged**